README.md

```diff
@@ -108,6 +108,21 @@ GoatRodeo.builder()
 | `--tempdir <value>`        | Directory for temporary storage (RAM disk recommended). |
 | `-t`, `--threads <value>`  | Number of threads (default: 4). Suggest 2–3× CPU cores. |
 | `-V`, `--version`          | Print version and exit. |
+| `--mime-filter`            | Allows artifacts to be filtered using a set of include |
+|                            | and exclude filters. The syntax of a filter is:        |
+|                            | `<command-char><filter-text>`                           |
+|                            | Command Characters:                                    |
+|                            | `+` include a mime type that matches exactly |
+|                            | `-` exclude a mime type that matches exactly |
+|                            | `*` include all mime types that match a regular expression |
+|                            | `/` exclude all mime types that match a regular expression |
+|                            | `#` ignore the filter text
```

Comment:
Oh that is gonna be funky on the command line, being the shell comment character.

Reply:
Sure, but you're more likely to need it in a `mime-filter-file` which uses exactly the same code path.

Reply:
yep. makes sense.